src/main/java/eiteam/esteemedinnovation/engineeringtable/GuiEngineeringTable.java

```diff
@@ -38,7 +41,9 @@ protected void drawGuiContainerBackgroundLayer(float partialTicks, int mouseX, i
         GL11.glEnable(3042);
         drawTexturedModalRect(k, l, 0, 0, xSize, ySize);
 
-        Item engineeringItem = tileEntity.getStackInSlot(0).getItem();
+        IItemHandler itemHandler = tileEntity.getCapability(ITEM_HANDLER_CAPABILITY, null);
+        assert itemHandler != null;
```

Comment:
You should not use assert except when testing. Change to an if statement, or an exception.